paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -1149,6 +1152,17 @@ public Path path() {
         return path(options.toMap());
     }
 
+    public String branch() {
+        return branch(options.toMap());
+    }
+
+    public static String branch(Map<String, String> options) {
+        if (options.containsKey(BRANCH.key())) {
```

Comment:
Just use options.get() will handle with the default value

Reply:
> Just use options.get() will handle with the default value

@schnappi17 Most of the parameters for this method come from "org. apache. paimon. table. FileStoreTable # options"

Reply:
> Just use options.get() will handle with the default value

@schnappi17 Most of the parameters for this method come from "org.apache.paimon.table.AbstractFileStoreTable #coreOptions, 'branch' parameter may be empty.

Reply:
I don't get why...

Reply:
Sorry, the previous description was incorrect,   Most of the parameters for this method come from "org.apache.paimon.table.FileStoreTable #options"， 'branch' parameter may be empty.